diff --git wysiwyg/wysiwyg.features.inc wysiwyg/wysiwyg.features.inc
index 5edd642..3f93160 100644
--- wysiwyg/wysiwyg.features.inc
+++ wysiwyg/wysiwyg.features.inc
@@ -52,7 +52,7 @@ function wysiwyg_features_export_render($module, $data, $export = NULL) {

   foreach ($data as $name) {
     if ($profile = wysiwyg_get_profile($name)) {
-      $profile_export = features_var_export($profile, '  ');
+      $profile_export = features_var_export($profile, "  /* $name */ ");
       $profile_identifier = features_var_export($profile->format);
       $code[] = "  // Exported profile: {$profile->format}";
       $code[] = "  \$profiles[{$profile_identifier}] = {$profile_export};";
